code/modules/atmospherics/components/binary_devices/circulator.dm

```diff
@@ -89,6 +89,11 @@
 
 /obj/machinery/atmospherics/binary/circulator/attackby(obj/item/weapon/W as obj, mob/user as mob)
 	if(isWrench(W))
+		var/air1pressure = air1.return_pressure()
+		var/air2pressure = air2.return_pressure()
+		if(air1pressure || air2pressure < ONE_ATMOSPHERE)
```

Comment:
Current reads as "if air1pressure is true (aka, exists) OR air2pressure is less than one atmosphere."
```suggestion
		if((air1pressure > ONE_ATMOSPHERE) || (air2pressure > ONE_ATMOSPHERE))
```

Reply:
Ah. I wasn't entirely sure on the use of ||.